### ISSUE TITLE
allow no cache headers for reproducible --offline mode

### ISSUE THREAD

**Issue by lizelive** [mamba-org/mamba#2881]
allowing the cache headers to be undefined and never expire would be very useful for repeatability
 
https://github.com/mamba-org/mamba/blob/474be8b9b0183c7d66f48d0ef516242fa85eba5f/libmamba/src/core/subdirdata.cpp#L283

an easy way to do this is to set default values here.

**Comment by jonashaag** [comment]
Please provide more context.

**Comment by lizelive** [comment]
yeah im trying to make a [nix](https://nixos.org/) builder for conda and using libmamba / micromamba to do so.

**Comment by jonashaag** [comment]
This still lacks a lot of concrete context for me to be able to understand the request, but tentatively marking as a feature request.
